## 03_nm_ppp_dialog.glade.patch.dpatch by Craig Box <ubuntu@dubculture.co.nz>
## 04_nm_ppp_properties.c.dpatch by Craig Box <ubuntu@dubculture.co.nz>

--- properties/nm-ppp-dialog.glade	(Revision 2743)
+++ properties/nm-ppp-dialog.glade	(Arbeitskopie)
@@ -286,305 +286,6 @@
 		    </packing>
 		  </child>
 
-		  <child>
-		    <widget class="GtkVBox" id="dialup-front">
-		      <property name="homogeneous">False</property>
-		      <property name="spacing">4</property>
-
-		      <child>
-			<widget class="GtkHBox" id="hbox20">
-			  <property name="visible">True</property>
-			  <property name="homogeneous">False</property>
-			  <property name="spacing">12</property>
-
-			  <child>
-			    <widget class="GtkLabel" id="label47">
-			      <property name="visible">True</property>
-			      <property name="label" translatable="yes">Telep_hone number:</property>
-			      <property name="use_underline">True</property>
-			      <property name="use_markup">False</property>
-			      <property name="justify">GTK_JUSTIFY_LEFT</property>
-			      <property name="wrap">False</property>
-			      <property name="selectable">False</property>
-			      <property name="xalign">1</property>
-			      <property name="yalign">0.5</property>
-			      <property name="xpad">0</property>
-			      <property name="ypad">0</property>
-			      <property name="mnemonic_widget">phone-number</property>
-			      <property name="ellipsize">PANGO_ELLIPSIZE_NONE</property>
-			      <property name="width_chars">-1</property>
-			      <property name="single_line_mode">False</property>
-			      <property name="angle">0</property>
-			    </widget>
-			    <packing>
-			      <property name="padding">0</property>
-			      <property name="expand">False</property>
-			      <property name="fill">False</property>
-			    </packing>
-			  </child>
-
-			  <child>
-			    <widget class="GtkEntry" id="phone-number">
-			      <property name="visible">True</property>
-			      <property name="tooltip" translatable="yes">Host name or IP address of the PPTP server</property>
-			      <property name="can_focus">True</property>
-			      <property name="editable">True</property>
-			      <property name="visibility">True</property>
-			      <property name="max_length">0</property>
-			      <property name="text" translatable="yes"></property>
-			      <property name="has_frame">True</property>
-			      <property name="invisible_char">*</property>
-			      <property name="activates_default">False</property>
-			    </widget>
-			    <packing>
-			      <property name="padding">0</property>
-			      <property name="expand">True</property>
-			      <property name="fill">True</property>
-			    </packing>
-			  </child>
-			</widget>
-			<packing>
-			  <property name="padding">0</property>
-			  <property name="expand">False</property>
-			  <property name="fill">False</property>
-			</packing>
-		      </child>
-		    </widget>
-		    <packing>
-		      <property name="padding">0</property>
-		      <property name="expand">True</property>
-		      <property name="fill">True</property>
-		    </packing>
-		  </child>
-
-		  <child>
-		    <widget class="GtkVBox" id="bluetooth-front">
-		      <property name="visible">True</property>
-		      <property name="homogeneous">False</property>
-		      <property name="spacing">6</property>
-
-		      <child>
-			<widget class="GtkTable" id="table99">
-			  <property name="visible">True</property>
-			  <property name="n_rows">2</property>
-			  <property name="n_columns">2</property>
-			  <property name="homogeneous">False</property>
-			  <property name="row_spacing">6</property>
-			  <property name="column_spacing">12</property>
-
-			  <child>
-			    <widget class="GtkLabel" id="label53">
-			      <property name="visible">True</property>
-			      <property name="label" translatable="yes">C_hannel:</property>
-			      <property name="use_underline">True</property>
-			      <property name="use_markup">False</property>
-			      <property name="justify">GTK_JUSTIFY_LEFT</property>
-			      <property name="wrap">False</property>
-			      <property name="selectable">False</property>
-			      <property name="xalign">0</property>
-			      <property name="yalign">0.5</property>
-			      <property name="xpad">0</property>
-			      <property name="ypad">0</property>
-			      <property name="mnemonic_widget">bt-channel</property>
-			      <property name="ellipsize">PANGO_ELLIPSIZE_NONE</property>
-			      <property name="width_chars">-1</property>
-			      <property name="single_line_mode">False</property>
-			      <property name="angle">0</property>
-			    </widget>
-			    <packing>
-			      <property name="left_attach">0</property>
-			      <property name="right_attach">1</property>
-			      <property name="top_attach">1</property>
-			      <property name="bottom_attach">2</property>
-			      <property name="x_options">fill</property>
-			      <property name="y_options"></property>
-			    </packing>
-			  </child>
-
-			  <child>
-			    <widget class="GtkEntry" id="bt-channel">
-			      <property name="visible">True</property>
-			      <property name="can_focus">True</property>
-			      <property name="editable">True</property>
-			      <property name="visibility">True</property>
-			      <property name="max_length">3</property>
-			      <property name="text" translatable="yes"></property>
-			      <property name="has_frame">True</property>
-			      <property name="invisible_char">*</property>
-			      <property name="activates_default">False</property>
-			    </widget>
-			    <packing>
-			      <property name="left_attach">1</property>
-			      <property name="right_attach">2</property>
-			      <property name="top_attach">1</property>
-			      <property name="bottom_attach">2</property>
-			      <property name="y_options"></property>
-			    </packing>
-			  </child>
-
-			  <child>
-			    <widget class="GtkEntry" id="bt-bdaddr">
-			      <property name="visible">True</property>
-			      <property name="tooltip" translatable="yes">Host name or IP address of the PPTP server</property>
-			      <property name="can_focus">True</property>
-			      <property name="editable">True</property>
-			      <property name="visibility">True</property>
-			      <property name="max_length">17</property>
-			      <property name="text" translatable="yes">00:00:00:00</property>
-			      <property name="has_frame">True</property>
-			      <property name="invisible_char">*</property>
-			      <property name="activates_default">False</property>
-			    </widget>
-			    <packing>
-			      <property name="left_attach">1</property>
-			      <property name="right_attach">2</property>
-			      <property name="top_attach">0</property>
-			      <property name="bottom_attach">1</property>
-			      <property name="y_options"></property>
-			    </packing>
-			  </child>
-
-			  <child>
-			    <widget class="GtkHBox" id="hbox21">
-			      <property name="visible">True</property>
-			      <property name="homogeneous">False</property>
-			      <property name="spacing">12</property>
-
-			      <child>
-				<widget class="GtkLabel" id="label49">
-				  <property name="visible">True</property>
-				  <property name="label" translatable="yes">_Device address:</property>
-				  <property name="use_underline">True</property>
-				  <property name="use_markup">False</property>
-				  <property name="justify">GTK_JUSTIFY_LEFT</property>
-				  <property name="wrap">False</property>
-				  <property name="selectable">False</property>
-				  <property name="xalign">1</property>
-				  <property name="yalign">0.5</property>
-				  <property name="xpad">0</property>
-				  <property name="ypad">0</property>
-				  <property name="mnemonic_widget">bt-bdaddr</property>
-				  <property name="ellipsize">PANGO_ELLIPSIZE_NONE</property>
-				  <property name="width_chars">-1</property>
-				  <property name="single_line_mode">False</property>
-				  <property name="angle">0</property>
-				</widget>
-				<packing>
-				  <property name="padding">0</property>
-				  <property name="expand">False</property>
-				  <property name="fill">False</property>
-				</packing>
-			      </child>
-			    </widget>
-			    <packing>
-			      <property name="left_attach">0</property>
-			      <property name="right_attach">1</property>
-			      <property name="top_attach">0</property>
-			      <property name="bottom_attach">1</property>
-			      <property name="x_options">fill</property>
-			      <property name="y_options">fill</property>
-			    </packing>
-			  </child>
-			</widget>
-			<packing>
-			  <property name="padding">0</property>
-			  <property name="expand">True</property>
-			  <property name="fill">True</property>
-			</packing>
-		      </child>
-
-		      <child>
-			<widget class="GtkHButtonBox" id="hbuttonbox2">
-			  <property name="visible">True</property>
-			  <property name="layout_style">GTK_BUTTONBOX_END</property>
-			  <property name="spacing">0</property>
-
-			  <child>
-			    <widget class="GtkButton" id="bt-find-device">
-			      <property name="visible">True</property>
-			      <property name="can_focus">True</property>
-			      <property name="relief">GTK_RELIEF_NORMAL</property>
-			      <property name="focus_on_click">True</property>
-
-			      <child>
-				<widget class="GtkAlignment" id="alignment28">
-				  <property name="visible">True</property>
-				  <property name="xalign">0.5</property>
-				  <property name="yalign">0.5</property>
-				  <property name="xscale">0</property>
-				  <property name="yscale">0</property>
-				  <property name="top_padding">0</property>
-				  <property name="bottom_padding">0</property>
-				  <property name="left_padding">0</property>
-				  <property name="right_padding">0</property>
-
-				  <child>
-				    <widget class="GtkHBox" id="hbox32">
-				      <property name="visible">True</property>
-				      <property name="homogeneous">False</property>
-				      <property name="spacing">2</property>
-
-				      <child>
-					<widget class="GtkImage" id="image2">
-					  <property name="visible">True</property>
-					  <property name="stock">gtk-find</property>
-					  <property name="icon_size">4</property>
-					  <property name="xalign">0.5</property>
-					  <property name="yalign">0.5</property>
-					  <property name="xpad">0</property>
-					  <property name="ypad">0</property>
-					</widget>
-					<packing>
-					  <property name="padding">0</property>
-					  <property name="expand">False</property>
-					  <property name="fill">False</property>
-					</packing>
-				      </child>
-
-				      <child>
-					<widget class="GtkLabel" id="label92">
-					  <property name="visible">True</property>
-					  <property name="label" translatable="yes">Find Device</property>
-					  <property name="use_underline">True</property>
-					  <property name="use_markup">False</property>
-					  <property name="justify">GTK_JUSTIFY_LEFT</property>
-					  <property name="wrap">False</property>
-					  <property name="selectable">False</property>
-					  <property name="xalign">0.5</property>
-					  <property name="yalign">0.5</property>
-					  <property name="xpad">0</property>
-					  <property name="ypad">0</property>
-					  <property name="ellipsize">PANGO_ELLIPSIZE_NONE</property>
-					  <property name="width_chars">-1</property>
-					  <property name="single_line_mode">False</property>
-					  <property name="angle">0</property>
-					</widget>
-					<packing>
-					  <property name="padding">0</property>
-					  <property name="expand">False</property>
-					  <property name="fill">False</property>
-					</packing>
-				      </child>
-				    </widget>
-				  </child>
-				</widget>
-			      </child>
-			    </widget>
-			  </child>
-			</widget>
-			<packing>
-			  <property name="padding">0</property>
-			  <property name="expand">True</property>
-			  <property name="fill">True</property>
-			</packing>
-		      </child>
-		    </widget>
-		    <packing>
-		      <property name="padding">0</property>
-		      <property name="expand">True</property>
-		      <property name="fill">True</property>
-		    </packing>
-		  </child>
 		</widget>
 		<packing>
 		  <property name="padding">0</property>
@@ -711,226 +412,6 @@
 	  </child>
 
 	  <child>
-	    <widget class="GtkTable" id="gprs-options">
-	      <property name="border_width">12</property>
-	      <property name="visible">True</property>
-	      <property name="n_rows">4</property>
-	      <property name="n_columns">2</property>
-	      <property name="homogeneous">False</property>
-	      <property name="row_spacing">6</property>
-	      <property name="column_spacing">12</property>
-
-	      <child>
-		<widget class="GtkLabel" id="label54">
-		  <property name="visible">True</property>
-		  <property name="label" translatable="yes">_Access point name:</property>
-		  <property name="use_underline">True</property>
-		  <property name="use_markup">False</property>
-		  <property name="justify">GTK_JUSTIFY_LEFT</property>
-		  <property name="wrap">False</property>
-		  <property name="selectable">False</property>
-		  <property name="xalign">1</property>
-		  <property name="yalign">0.5</property>
-		  <property name="xpad">0</property>
-		  <property name="ypad">0</property>
-		  <property name="mnemonic_widget">gprs-apn</property>
-		  <property name="ellipsize">PANGO_ELLIPSIZE_NONE</property>
-		  <property name="width_chars">-1</property>
-		  <property name="single_line_mode">False</property>
-		  <property name="angle">0</property>
-		</widget>
-		<packing>
-		  <property name="left_attach">0</property>
-		  <property name="right_attach">1</property>
-		  <property name="top_attach">0</property>
-		  <property name="bottom_attach">1</property>
-		  <property name="x_options">fill</property>
-		  <property name="y_options"></property>
-		</packing>
-	      </child>
-
-	      <child>
-		<widget class="GtkEntry" id="gprs-apn">
-		  <property name="visible">True</property>
-		  <property name="tooltip" translatable="yes">Service providers GPRS access point from device config</property>
-		  <property name="can_focus">True</property>
-		  <property name="editable">True</property>
-		  <property name="visibility">True</property>
-		  <property name="max_length">0</property>
-		  <property name="text" translatable="yes"></property>
-		  <property name="has_frame">True</property>
-		  <property name="invisible_char">*</property>
-		  <property name="activates_default">False</property>
-		</widget>
-		<packing>
-		  <property name="left_attach">1</property>
-		  <property name="right_attach">2</property>
-		  <property name="top_attach">0</property>
-		  <property name="bottom_attach">1</property>
-		  <property name="y_options"></property>
-		</packing>
-	      </child>
-
-	      <child>
-		<widget class="GtkLabel" id="label57">
-		  <property name="visible">True</property>
-		  <property name="label" translatable="yes">IP a_ddress:</property>
-		  <property name="use_underline">True</property>
-		  <property name="use_markup">False</property>
-		  <property name="justify">GTK_JUSTIFY_LEFT</property>
-		  <property name="wrap">False</property>
-		  <property name="selectable">False</property>
-		  <property name="xalign">0</property>
-		  <property name="yalign">0.5</property>
-		  <property name="xpad">0</property>
-		  <property name="ypad">0</property>
-		  <property name="mnemonic_widget">gprs-ip-address</property>
-		  <property name="ellipsize">PANGO_ELLIPSIZE_NONE</property>
-		  <property name="width_chars">-1</property>
-		  <property name="single_line_mode">False</property>
-		  <property name="angle">0</property>
-		</widget>
-		<packing>
-		  <property name="left_attach">0</property>
-		  <property name="right_attach">1</property>
-		  <property name="top_attach">1</property>
-		  <property name="bottom_attach">2</property>
-		  <property name="x_options">fill</property>
-		  <property name="y_options"></property>
-		</packing>
-	      </child>
-
-	      <child>
-		<widget class="GtkEntry" id="gprs-ip-address">
-		  <property name="visible">True</property>
-		  <property name="tooltip" translatable="yes">Service providers IP address</property>
-		  <property name="can_focus">True</property>
-		  <property name="editable">True</property>
-		  <property name="visibility">True</property>
-		  <property name="max_length">0</property>
-		  <property name="text" translatable="yes"></property>
-		  <property name="has_frame">True</property>
-		  <property name="invisible_char">*</property>
-		  <property name="activates_default">False</property>
-		</widget>
-		<packing>
-		  <property name="left_attach">1</property>
-		  <property name="right_attach">2</property>
-		  <property name="top_attach">1</property>
-		  <property name="bottom_attach">2</property>
-		  <property name="y_options"></property>
-		</packing>
-	      </child>
-
-	      <child>
-		<widget class="GtkLabel" id="label58">
-		  <property name="visible">True</property>
-		  <property name="label" translatable="yes">_Packet type:</property>
-		  <property name="use_underline">True</property>
-		  <property name="use_markup">False</property>
-		  <property name="justify">GTK_JUSTIFY_LEFT</property>
-		  <property name="wrap">False</property>
-		  <property name="selectable">False</property>
-		  <property name="xalign">0</property>
-		  <property name="yalign">0.5</property>
-		  <property name="xpad">0</property>
-		  <property name="ypad">0</property>
-		  <property name="mnemonic_widget">gprs-packet-type</property>
-		  <property name="ellipsize">PANGO_ELLIPSIZE_NONE</property>
-		  <property name="width_chars">-1</property>
-		  <property name="single_line_mode">False</property>
-		  <property name="angle">0</property>
-		</widget>
-		<packing>
-		  <property name="left_attach">0</property>
-		  <property name="right_attach">1</property>
-		  <property name="top_attach">2</property>
-		  <property name="bottom_attach">3</property>
-		  <property name="x_options">fill</property>
-		  <property name="y_options"></property>
-		</packing>
-	      </child>
-
-	      <child>
-		<widget class="GtkEntry" id="gprs-packet-type">
-		  <property name="visible">True</property>
-		  <property name="can_focus">True</property>
-		  <property name="editable">True</property>
-		  <property name="visibility">True</property>
-		  <property name="max_length">0</property>
-		  <property name="text" translatable="yes"></property>
-		  <property name="has_frame">True</property>
-		  <property name="invisible_char">*</property>
-		  <property name="activates_default">False</property>
-		  <property name="width_chars">9</property>
-		</widget>
-		<packing>
-		  <property name="left_attach">1</property>
-		  <property name="right_attach">2</property>
-		  <property name="top_attach">2</property>
-		  <property name="bottom_attach">3</property>
-		  <property name="y_options"></property>
-		</packing>
-	      </child>
-
-	      <child>
-		<widget class="GtkLabel" id="label59">
-		  <property name="visible">True</property>
-		  <property name="label" translatable="yes">Context _number:</property>
-		  <property name="use_underline">True</property>
-		  <property name="use_markup">False</property>
-		  <property name="justify">GTK_JUSTIFY_LEFT</property>
-		  <property name="wrap">False</property>
-		  <property name="selectable">False</property>
-		  <property name="xalign">0</property>
-		  <property name="yalign">0.5</property>
-		  <property name="xpad">0</property>
-		  <property name="ypad">0</property>
-		  <property name="mnemonic_widget">gprs-context-num</property>
-		  <property name="ellipsize">PANGO_ELLIPSIZE_NONE</property>
-		  <property name="width_chars">-1</property>
-		  <property name="single_line_mode">False</property>
-		  <property name="angle">0</property>
-		</widget>
-		<packing>
-		  <property name="left_attach">0</property>
-		  <property name="right_attach">1</property>
-		  <property name="top_attach">3</property>
-		  <property name="bottom_attach">4</property>
-		  <property name="x_options">fill</property>
-		  <property name="y_options"></property>
-		</packing>
-	      </child>
-
-	      <child>
-		<widget class="GtkEntry" id="gprs-context-num">
-		  <property name="visible">True</property>
-		  <property name="can_focus">True</property>
-		  <property name="editable">True</property>
-		  <property name="visibility">True</property>
-		  <property name="max_length">0</property>
-		  <property name="text" translatable="yes"></property>
-		  <property name="has_frame">True</property>
-		  <property name="invisible_char">*</property>
-		  <property name="activates_default">False</property>
-		  <property name="width_chars">3</property>
-		</widget>
-		<packing>
-		  <property name="left_attach">1</property>
-		  <property name="right_attach">2</property>
-		  <property name="top_attach">3</property>
-		  <property name="bottom_attach">4</property>
-		  <property name="y_options"></property>
-		</packing>
-	      </child>
-	    </widget>
-	    <packing>
-	      <property name="tab_expand">False</property>
-	      <property name="tab_fill">True</property>
-	    </packing>
-	  </child>
-
-	  <child>
 	    <widget class="GtkLabel" id="label76">
 	      <property name="visible">True</property>
 	      <property name="label" translatable="yes">GPRS Options</property>
@@ -2075,80 +1556,6 @@
 	  </child>
 
 	  <child>
-	    <widget class="GtkVBox" id="serial-options">
-	      <property name="border_width">12</property>
-	      <property name="visible">True</property>
-	      <property name="homogeneous">False</property>
-	      <property name="spacing">6</property>
-
-	      <child>
-		<widget class="GtkCheckButton" id="ppp-modem">
-		  <property name="visible">True</property>
-		  <property name="can_focus">True</property>
-		  <property name="label" translatable="yes">_Modem connection</property>
-		  <property name="use_underline">True</property>
-		  <property name="relief">GTK_RELIEF_NORMAL</property>
-		  <property name="focus_on_click">True</property>
-		  <property name="active">False</property>
-		  <property name="inconsistent">False</property>
-		  <property name="draw_indicator">True</property>
-		</widget>
-		<packing>
-		  <property name="padding">0</property>
-		  <property name="expand">False</property>
-		  <property name="fill">False</property>
-		</packing>
-	      </child>
-
-	      <child>
-		<widget class="GtkCheckButton" id="ppp-crtscts">
-		  <property name="visible">True</property>
-		  <property name="can_focus">True</property>
-		  <property name="label" translatable="yes">_Hardware RTS/CTS</property>
-		  <property name="use_underline">True</property>
-		  <property name="relief">GTK_RELIEF_NORMAL</property>
-		  <property name="focus_on_click">True</property>
-		  <property name="active">False</property>
-		  <property name="inconsistent">False</property>
-		  <property name="draw_indicator">True</property>
-		</widget>
-		<packing>
-		  <property name="padding">0</property>
-		  <property name="expand">False</property>
-		  <property name="fill">False</property>
-		</packing>
-	      </child>
-	    </widget>
-	    <packing>
-	      <property name="tab_expand">False</property>
-	      <property name="tab_fill">True</property>
-	    </packing>
-	  </child>
-
-	  <child>
-	    <widget class="GtkLabel" id="label80">
-	      <property name="visible">True</property>
-	      <property name="label" translatable="yes">Serial Options</property>
-	      <property name="use_underline">False</property>
-	      <property name="use_markup">False</property>
-	      <property name="justify">GTK_JUSTIFY_LEFT</property>
-	      <property name="wrap">False</property>
-	      <property name="selectable">False</property>
-	      <property name="xalign">0.5</property>
-	      <property name="yalign">0.5</property>
-	      <property name="xpad">0</property>
-	      <property name="ypad">0</property>
-	      <property name="ellipsize">PANGO_ELLIPSIZE_NONE</property>
-	      <property name="width_chars">-1</property>
-	      <property name="single_line_mode">False</property>
-	      <property name="angle">0</property>
-	    </widget>
-	    <packing>
-	      <property name="type">tab</property>
-	    </packing>
-	  </child>
-
-	  <child>
 	    <widget class="GtkVBox" id="routing-options">
 	      <property name="border_width">12</property>
 	      <property name="visible">True</property>
diff -urNad network-manager-pptp-0.6.4+pptp0.6.5~/properties/nm-ppp-properties.c network-manager-pptp-0.6.4+pptp0.6.5/properties/nm-ppp-properties.c
--- properties/nm-ppp-properties.c	2007-05-26 12:19:31.000000000 +1200
+++ properties/nm-ppp-properties.c	2007-05-26 12:18:27.000000000 +1200
@@ -42,7 +42,7 @@
 
 #define BUILD_BLUETOOTH
 
-#define VPNUI_DISPLAY_NAME _("pppd tunnel (PPTP, BTGPRS, Dialup)")
+#define VPNUI_DISPLAY_NAME _("PPTP tunnel")
 #define VPNUI_SERVICE_NAME "org.freedesktop.NetworkManager.ppp_starter"
 #define VPNUI_BASIC_DEFAULTS "connection-name='';" \
                              "ppp-debug=no;" \
@@ -50,7 +50,7 @@
                              "usepeerdns-overtunnel=yes;" \
                              "ppp-lock=yes;" \
                              "ppp-auth-peer=no;" \
-                             "ppp-refuse-eap=no;" \
+                             "ppp-refuse-eap=yes;" \
                              "ppp-refuse-chap=no;" \
                              "ppp-refuse-mschap=no;" \
                              "lcp-echo-failure=10;" \
@@ -59,26 +59,6 @@
                              "routes=;" \
                              "ppp-debug=no;" \
                              "ppp-extra='';"
-#define VPNUI_BTOOTH_DEFAULTS "bt-bdaddr=00:00:00:00:00:00;" \
-                              "bt-channel=1;" 
-#define VPNUI_GPRS_DEFAULTS "gprs-packet-type=IP;" \
-                             "gprs-context-num=1;" \
-                             "gprs-ip-address=0.0.0.0;" \
-                             "gprs-apn=internet;" \
-                             "ppp-crtscts=yes;" \
-                             "ppp-noipdefault=yes;" \
-                             "ppp-modem=yes;" \
-                             "usepeerdns=yes;" \
-                             "ppp-connect-delay=5000;" 
-#define VPNUI_BTGPRS_DEFAULTS "ppp-connection-type=btgprs;" \
-                             "encrypt-mppe=no;" \
-                             "encrypt-mppe-128=no;" \
-                             "encrypt-mppe-stateful=no;" \
-                             "compress-mppc=no;" \
-                             "compress-bsd=no;" \
-                             "compress-deflate=no;" \
-                             "mru=1000;" \
-                             "mtu=1000;" 
 #define VPNUI_PPTP_DEFAULTS "pptp-remote='';" \
                              "ppp-connection-type=pptp;" \
                              "encrypt-mppe=no;" \
@@ -89,18 +69,6 @@
                              "compress-deflate=no;" \
                              "mru=1416;" \
                              "mtu=1416;" 
-#define VPNUI_DIALUP_DEFAULTS "phone-number=THIS DOESN'T DO ANYTHING;" \
-                              "ppp-crtscts=yes;" \
-                              "ppp-modem=yes;" \
-                              "ppp-connection-type=dialup;" \
-                              "encrypt-mppe=no;" \
-                              "encrypt-mppe-128=no;" \
-                              "encrypt-mppe-stateful=no;" \
-                              "compress-mppc=no;" \
-                              "compress-bsd=no;" \
-                              "compress-deflate=no;" \
-                              "mru=1000;" \
-                              "mtu=1000;" 
 
 #ifdef NMVPNUI_PPTP_PROPERTIES_C
 #endif
@@ -192,51 +160,6 @@
   GTK_SIGNAL_FUNC(&editable_changed), NULL, impl );
 
   opt = vpnui_opt_new(
-  "phone-number", VPN_UI_OPTTYPE_STRING, 
-  "phone-number", "Telephone-Number", _("Telephone Number"),
-  GTK_SIGNAL_FUNC(&editable_changed), &vld_non_empty, impl );
-
-  opt = vpnui_opt_new(
-  "bt-bdaddr", VPN_UI_OPTTYPE_STRING, 
-  "bt-bdaddr", "Bluetooth-Address", _("Bluetooth Address"),
-  GTK_SIGNAL_FUNC(&editable_changed), &vld_non_empty, impl );
-
-  opt = vpnui_opt_new(
-  "bt-channel", VPN_UI_OPTTYPE_STRING, 
-  "bt-channel", "Bluetooth-Channel", _("Bluetooth Channel"),
-  GTK_SIGNAL_FUNC(&editable_changed), &vld_non_empty, impl );
-
-  opt = vpnui_opt_new(
-  "gprs-apn", VPN_UI_OPTTYPE_STRING, 
-  "gprs-apn", "GPRS-Access-Point-Name", _("GPRS APN"),
-  GTK_SIGNAL_FUNC(&editable_changed), &vld_non_empty, impl );
-
-  opt = vpnui_opt_new(
-  "gprs-ip-address", VPN_UI_OPTTYPE_STRING, 
-  "gprs-ip-address", "GPRS-IP-Address", _("GPRS IP"),
-  GTK_SIGNAL_FUNC(&editable_changed), &vld_non_empty, impl );
-
-  opt = vpnui_opt_new(
-  "gprs-context-num", VPN_UI_OPTTYPE_STRING, 
-  "gprs-context-num", "GPRS-Context-Number", _("GPRS Context No."),
-  GTK_SIGNAL_FUNC(&editable_changed), &vld_non_empty, impl );
-
-  opt = vpnui_opt_new(
-  "gprs-packet-type", VPN_UI_OPTTYPE_STRING, 
-  "gprs-packet-type", "GPRS-Packet-Type", _("GPRS Packet Type"),
-  GTK_SIGNAL_FUNC(&editable_changed), &vld_non_empty, impl );
-
-  opt = vpnui_opt_new(
-  "ppp-crtscts", VPN_UI_OPTTYPE_YESNO, 
-  "ppp-crtscts", "PPP-Hardware-CTSRTS", _("Use CTS/RTS flow control"),
-  GTK_SIGNAL_FUNC(&editable_changed), NULL, impl );
-
-  opt = vpnui_opt_new(
-  "ppp-modem", VPN_UI_OPTTYPE_YESNO, 
-  "ppp-modem", "PPP-Modem", _("Connect via a modem"),
-  GTK_SIGNAL_FUNC(&editable_changed), NULL, impl );
-
-  opt = vpnui_opt_new(
   "ppp-noipdefault", VPN_UI_OPTTYPE_YESNO, 
   "ppp-noipdefault", "PPP-No-IP-Default", _("Require IP to be provided"),
   GTK_SIGNAL_FUNC(&editable_changed), NULL, impl );
@@ -358,29 +281,14 @@
                         VPNUI_BASIC_DEFAULTS VPNUI_PPTP_DEFAULTS,
                         impl);
 
-#ifdef BUILD_BLUETOOTH
-  variant = vpnui_variant_new( "btgprs", "GPRS over Bluetooth (via mobile phone)",
-                        VPNUI_BASIC_DEFAULTS VPNUI_GPRS_DEFAULTS VPNUI_BTOOTH_DEFAULTS VPNUI_BTGPRS_DEFAULTS,
-                        impl);
-#endif
+  gtk_combo_box_set_active(impl->variant_combo, 0);
+//  vpnui_variant_select_byname(impl,"Windows VPN (PPTP)");
 
-  variant = vpnui_variant_new( "dialup", "Dialup",
-                        VPNUI_BASIC_DEFAULTS VPNUI_DIALUP_DEFAULTS,
-                        impl);
 //
 //                     GLADE NAME   IMPLEMENTATION_OBJ
 //  expand= vpnui_expand_new ("routing-expander",impl);
-//  expand= vpnui_expand_new ("dialup-expander",impl);
 //  expand= vpnui_expand_new ("pppd-expander",impl);
 //  expand= vpnui_expand_new ("pptp-expander",impl);
-//  expand= vpnui_expand_new ("bluetooth-expander",impl);
-//  expand= vpnui_expand_new ("serial-expander",impl);
-//  expand= vpnui_expand_new ("gprs-expander",impl);
-
-// Attach to press event of the Bluetooth "Find Device" button.
-//    will need libbtcl
-//  impl->w_import_button = GTK_BUTTON (glade_xml_get_widget (impl->xml, 
-//								       "import-button"));
 
 // Attach an import_button
   impl->w_import_button = GTK_BUTTON (glade_xml_get_widget (impl->xml, 
@@ -444,15 +352,11 @@
 void 
 impl_hide_and_show (NetworkManagerVpnUIImpl *impl)
 {
-  GtkWidget *serial_options = NULL;
-  GtkWidget *gprs_options = NULL;
   GtkWidget *auth_options = NULL;
   GtkWidget *comp_enc_options = NULL;
   GtkWidget *routing_options = NULL;
   GtkWidget *ppp_options = NULL;
-  GtkWidget *bluetooth_front = NULL;
   GtkWidget *pptp_front = NULL;
-  GtkWidget *dialup_front = NULL;
   GtkComboBox *combo = NULL;
   char *variant_name;
 
@@ -466,17 +370,9 @@
   auth_options = glade_xml_get_widget(impl->xml, "auth-options");
   comp_enc_options = glade_xml_get_widget(impl->xml, "comp-enc-options");
   routing_options = glade_xml_get_widget(impl->xml, "routing-options");
-  gprs_options = glade_xml_get_widget(impl->xml, "gprs-options");
-  serial_options = glade_xml_get_widget(impl->xml, "serial-options");
-  bluetooth_front = glade_xml_get_widget(impl->xml, "bluetooth-front");
-  dialup_front = glade_xml_get_widget(impl->xml, "dialup-front");
   pptp_front = glade_xml_get_widget(impl->xml, "pptp-front");
 
   if (pptp_front) gtk_widget_hide(pptp_front);
-  if (bluetooth_front) gtk_widget_hide(bluetooth_front);
-  if (dialup_front) gtk_widget_hide(dialup_front);
-  if (gprs_options) gtk_widget_hide(gprs_options);
-  if (serial_options) gtk_widget_hide(serial_options);
   if (ppp_options) gtk_widget_hide(ppp_options);
   if (auth_options) gtk_widget_hide(auth_options);
   if (comp_enc_options) gtk_widget_hide(comp_enc_options);
@@ -491,20 +387,5 @@
     if (auth_options)     gtk_widget_show(auth_options);
     if (comp_enc_options) gtk_widget_show(comp_enc_options);
     if (routing_options)  gtk_widget_show(routing_options);
-  } else if (strcmp(variant_name,"dialup")==0) {
-    if (dialup_front) gtk_widget_show(dialup_front);
-    if (serial_options) gtk_widget_show(serial_options);
-    if (ppp_options)      gtk_widget_show(ppp_options);
-    if (auth_options)     gtk_widget_show(auth_options);
-    if (comp_enc_options) gtk_widget_show(comp_enc_options);
-    if (routing_options)  gtk_widget_show(routing_options);
-  } else if (strcmp(variant_name,"btgprs")==0) {
-    if (bluetooth_front) gtk_widget_show(bluetooth_front);
-    if (gprs_options) gtk_widget_show(gprs_options);
-    if (serial_options) gtk_widget_show(serial_options);
-    if (ppp_options)      gtk_widget_show(ppp_options);
-    if (auth_options)     gtk_widget_show(auth_options);
-    if (comp_enc_options) gtk_widget_show(comp_enc_options);
-    if (routing_options)  gtk_widget_show(routing_options);
   }  
 }
